pkg/sql/logictest/testdata/logic_test/tenant

```diff
@@ -550,3 +550,26 @@ RESET CLUSTER SETTING server.controller.default_target_cluster
 
 statement ok
 DROP TENANT withservice
+
+subtest restrict_system_access
+
+statement ok
+SET CLUSTER SETTING sql.restrict_system_interface.enabled = true
+
+statement error blocked update to application-level setting.*\nHINT:.*\nTry changing the setting from a virtual cluster
+SET CLUSTER SETTING ui.display_timezone = 'America/New_York'
```

Comment:
This is an interesting example because on the destination cluster where operators will be managing their physical replication job, perhaps they do what to set this.

However, in that case, they can flip sql.restrict_system_interface.enabled off and back on.